nextcord/embeds.py

```diff
@@ -616,13 +616,18 @@ def insert_field_at(self: E, index: int, *, name: Any, value: Any, inline: bool
 
         return self
 
-    def clear_fields(self) -> None:
-        """Removes all fields from this embed."""
+    def clear_fields(self: E) -> E:
+        """Removes all fields from this embed.
+
+        This function returns the class instance to allow for fluent-style
+        chaining.
```

Comment:
Small typo missing end quotes.

```suggestion
        chaining.
        """
```

Also, `remove_field` could return self as well maybe?